tests/tests-asn1c-compiler/13-resolver-OK.asn1.-B

```diff
@@ -0,0 +1,29 @@
+////////////////////// moduletestresolver1.proto //////////////////////
+// Protobuf generated from /13-resolver-OK.asn1 by asn1c-0.9.29
+// ModuleTestResolver1 { iso org(3) dod(6) internet(1) private(4) enterprise(1) spelio(9363) software(1) asn1c(5) test(1) 13 1 }
+
+syntax = "proto3";
+
+package pkg13_resolver_ok_asn1.v1;
+
+import "pkg13_resolver_ok_asn1/v1/moduletestresolver2.proto"; // { iso org(3) dod(6) internet(1) private(4) enterprise(1) spelio(9363) software(1) asn1c(5) test(1) 13 2 }
```

Comment:
package should be `moduletestresolver2.proto`, but it's generated as `renamed.proto`